script.module.clouddrive.common/README.md

```diff
@@ -0,0 +1,11 @@
+# script.module.clouddrive.common
+
+
+## Usage
```

Comment:
Could you expand on the modules usage? Is it primarily for https://github.com/xbmc/repo-plugins/pull/1508

Documentation for other developers could be useful.

Reply:
It's primarily for xbmc/repo-plugins#1493 and xbmc/repo-plugins#1508. And I have 2 more coming (Dropbox and Box).
I'll add a manual later in case somebody wants to add another cloud provider. 
But is this a blocker? Hope not.

Reply:
No it's not a block... The module should provide some data to aid any potential developers seeking to use it.... You can add one later :)